ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -100,24 +178,29 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
         </TableCell>
 
         <TableCell>
-          <TableCellListItem
-            button
-            component={Link}
-            to={`${path}/providers/${workerPool.providerId}`}>
-            <ListItemText
-              disableTypography
-              primary={<Typography>{workerPool.providerId}</Typography>}
-            />
-            <LinkIcon size={iconSize} />
-          </TableCellListItem>
+          <Typography>
+            {workerPool.providerId !== NULL_PROVIDER && workerPool.providerId}
+          </Typography>
+        </TableCell>
+
+        <TableCell>
+          {workerPool.providerId !== NULL_PROVIDER && (
+            <IconButton
+              className={classNames(classes.button, classes.deleteButton)}
+              name={`${workerPool.workerPoolId}`}
+              onClick={this.handleDeleteClick}
+              disabled={actionLoading}>
+              <DeleteIcon size={iconSize} />
+            </IconButton>
+          )}
```

Comment:
![Screen Shot 2019-07-03 at 10 23 21 AM](https://user-images.githubusercontent.com/3766511/60599560-acac7680-9d7c-11e9-8d14-40b4815615a0.png)

The delete icon button looks different in the light theme. I still think https://github.com/taskcluster/taskcluster/pull/865#discussion_r297657449 is valid here and aligns better with the material design spec that is mentioned in https://github.com/taskcluster/taskcluster/pull/922#discussion_r299955275.